sentinel-core/src/main/java/com/alibaba/csp/sentinel/context/ContextUtil.java

```diff
@@ -49,13 +49,33 @@
     private static ThreadLocal<Context> contextHolder = new ThreadLocal<Context>();
 
     /**
-     * Holds all {@link EntranceNode}
+     * Holds all {@link EntranceNode}.
      */
     private static volatile Map<String, DefaultNode> contextNameNodeMap = new HashMap<String, DefaultNode>();
 
     private static final ReentrantLock LOCK = new ReentrantLock();
     private static final Context NULL_CONTEXT = new NullContext();
 
+    static {
+        // Cache the entrance node for default context.
+        initDefaultContext();
+    }
+
+    private static void initDefaultContext() {
+        contextNameNodeMap.put(Constants.CONTEXT_DEFAULT_NAME, new EntranceNode(
```

Comment:
*Default Context* should be child of *ROOT*,  use `Constants.ROOT.addChild(node)` to add.